features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1561,6 +1561,8 @@
   "connector.challenge_questions.enable": false,
   "tenant_mgt.enable_email_domain": false,
 
-  "on_demand_config.on_initial_use.enable_sms_otp_password_recovery_if_connector_enabled": false
+  "on_demand_config.on_initial_use.enable_sms_otp_password_recovery_if_connector_enabled": false,
```

Comment:
This seems to be something removed. Do we have to add it back ?

Reply:
Diff appears due to the addition of comma `,` to add the below config related to actions.